app/model/ContentSubscriptionPlanOps.scala

```diff
@@ -2,7 +2,7 @@ package model
 
 import com.gu.i18n.Currency._
 import com.gu.i18n._
-import com.gu.memsub.BillingPeriod.{OneYear, Quarter, SixWeeks, Year}
+import com.gu.memsub.BillingPeriod.{OneYear, Quarter, SixWeeks, ThreeMonths, Year}
```

Comment:
Should the finite ones be named so as to make it totally clear? Something like 'ThreeMonthsFinite' or 'ThreeMonthsGift' or 'ThreeMonthsOnly'?

Reply:
That would be a massive change unfortunately as these are defined in membership-common which is a separate library and is consumed by a lot of other projects

Reply:
👍